torch/_subclasses/functional_tensor.py

```diff
@@ -533,6 +546,20 @@ def unwrap(x):
                         torch.Tensor, wrap, outs_unwrapped
                     )
                 else:
+                    # Check args before the operation
+                    for arg in args_unwrapped:
```

Comment:
I think we discussed this before but i forgot hahah, how can this work when we don't have a proper functionlization rule for aten._to_copy that always aliases.

Reply:
> I think we discussed this before but i forgot hahah, how can this work when we don't have a proper functionlization rule for aten._to_copy that always aliases.

again, not sure if this is a comment on the outdated PR. I think we have a proper functionlization rule for aten._to_copy  now? register_aten_to_copy_py_functionalize_impl